internal/backends/postgresql/postgresql.go

```diff
@@ -80,6 +71,16 @@ func collectionsStats(ctx context.Context, p *pgxpool.Pool, dbName string, list
 	// initialization fork https://www.postgresql.org/docs/current/storage-init.html,
 	// free space map https://www.postgresql.org/docs/current/storage-fsm.html and
 	// TOAST https://www.postgresql.org/docs/current/storage-toast.html.
+
```

Comment:
The code that calculates stats should be there, not in `AnalyzeColl`. That looks like a bad merge